packages/lisk-p2p/test/integration/p2p.ts

```diff
@@ -151,11 +150,10 @@ describe('Integration tests for P2P library', () => {
 		beforeEach(async () => {
 			p2pNodeList = [...Array(NETWORK_PEER_COUNT).keys()].map(index => {
 				// Each node will have the next node in the sequence as a seed peer.
-				const seedPeers = [
+				const seedPeers: ReadonlyArray<P2PPeerInfo> = [
```

Comment:
do we need to define here? same as below

Reply:
I just added to use autocompletion to fill up the field names, but yeah we don't need it, I'll remove it.